thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -387,16 +387,16 @@ def train(self):
             if i == self.warmup_iter:  # warmup
                 t0 = iter_t0
 
+            if self.nsys_enabled and i == self.profiler_start and global_rank in [0, None] and step_idx == 0:
```

Comment:
We should remove the last `step_idx == 0` as it is undefined at this point.

Reply:
Ah! I got distracted with other tasks yesterday and forgot to run the code. Thanks for catching this. Fixed.